db/migrate/20220217210814_update_reporting_patient_states_to_version_5.rb

```diff
@@ -0,0 +1,21 @@
+class UpdateReportingPatientStatesToVersion5 < ActiveRecord::Migration[5.2]
+  def change
```

Comment:
This migration probably needs some work, because we have to ensure any dependent views are dropped and then recreated after the `patient_states` change, and also have to make sure all indexes are recreated.